packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/xcpd-request.ts

```diff
@@ -72,6 +76,10 @@ export function processInboundXcpdRequest(request: string): InboundPatientDiscov
         iti55Request.Envelope.Header.Security.Signature.SignatureValue
       ),
     };
+
+    await storeXcpdRequest({ request, inboundRequest });
```

Comment:
we are storing after processing. If it errors we wont store it but we will have logs to look at. ditto for the rest

Reply:
ya that's good with me